crates/cast/src/starknet_commands/account/mod.rs

```diff
@@ -135,7 +138,16 @@ pub fn add_created_profile_to_configuration(
             Value::String(config.accounts_file.to_string()),
         );
 
-        tool_sncast.insert(config.account.clone(), Value::Table(new_profile));
```

Comment:
should accounts-file be present in the toml file if we used keystore? I think that, unless specifically added by multiple, various account create commands, we should only add what's needed to toml file (so eg no keystore when creating "classic" account, and probably no accounts-file otherwise)